sequence/src/main/java/com/vitreoussoftware/bioinformatics/sequence/encoding/BasicDnaEncodingScheme.java

```diff
@@ -10,14 +10,16 @@
  * @author John
  *
  */
-    public final class BasicDnaEncodingScheme implements EncodingScheme {
+public final class BasicDnaEncodingScheme implements EncodingScheme {
     // Due to a quirk in the java language we have to use the negative sign to set the 8th bit to 1
     // We will restrict usage of the 8th bit to mean 'special thing happening here
 	private static final byte NUCLEOTIDE_A =  0b0_00_00_10;
 	private static final byte NUCLEOTIDE_C =  0b0_00_10_00;
 	private static final byte NUCLEOTIDE_G =  0b0_00_01_00;
 	private static final byte NUCLEOTIDE_T =  0b0_00_00_01;
 
+	private static final byte SHIFT_RIGHT = NUCLEOTIDE_A | NUCLEOTIDE_C;
```

Comment:
Should potentially be named a little more exactly: SHIFT_RIGHT_TO_COMPLEMENT or something.  Or maybe just a comment.  When I first read it, I asked myself how this value would be used to perform a right shift.